google/cloud/bigquery/storage_v1/__init__.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from .services.big_query_read import BigQueryReadClient
```

Comment:
Let's have this be the manual wrapper.

Reply:
This is a generated file and `bigquery_storage_v1/client.py` imports from it. The latter assumes that the generated client  is imported and wraps it, although it could also import directly from `storage_v1.services.big_query_read`, in principle.

Do you think it's worth maintaining additional synth replacement rules to prevent users from accessing the generated client through `google.cloud.bigquery.storage_v1`?

Reply:
> Do you think it's worth maintaining additional synth replacement rules to prevent users from accessing the generated client through google.cloud.bigquery.storage_v1?

I'm pretty sure that's what I did for the `bigquery_storage_v1beta1` package (that is, exclude the equivalent top-level `__init__.py` from getting copied over.

I'm still flip-flopping about what namespace we should use for the manual clients. It's nice to avoid breaking changes, but if we're breaking anyway, maybe we should be consistent. What do other manual libs do for namespace of manual clients?

Reply:
PubSub just exposes the generated clients through `pubsub_v1.__init__.py` [import](https://github.com/googleapis/python-pubsub/blob/master/google/pubsub_v1/__init__.py#L18-L19).

I don't know about other libs, but can check (I suspect they left these imports intact, though).

Reply:
Interesting! I find that it uses the `google.pubsub_v1` for generated and `google.cloud.pubsub_v1` for handwritten to be pretty confusing, but I guess so long as it's documented well, it'll be okay.

Reply:
I can at least mention this in the UPGRADING guide and just recommend to import from `google.cloud.bigquery.storage` (the samples use this path, too). This will give us more freedom to restructure the library internally in the future (in theory at least).